### PR TITLE
Feat(kclvm-win): remove the "kclvm.requirements.done.txt" on windows.

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

issue #379.

#### 2. What is the scope of this PR (e.g. component or file name):

kclvm windows.

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

1. Removed the process of using the flag file to determine if kclvm is successfully installed, and instead using the command "python3 -m pip show kclvm" to determine if kclvm is successfully installed.

2.  Add the -U parameter in all "pip install" commands to ensure the latest version of kclvm is obtained.

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [x] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by NeverRaR — DISMISSED** [review]
lgtm

File: scripts/build-windows/kcl-doc.go [hunk 1]
```diff
@@ -57,29 +57,22 @@ func Install_Kclvm(installed_path string) {
...
 
-	// Check if "installed" file exists
-	outputPath := filepath.Join(installed_path, "kclvm_installed")
-	if _, err := os.Stat(outputPath); err == nil {
+	cmd = exec.Command("cmd", "/C", "python3", "-m", "pip", "show", "kclvm")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+
+	// Check if kclvm has been installed.
+	if err := cmd.Run(); err == nil && out.String() != "WARNING: Package(s) not found: kclvm" {
```
Comment (Peefy) [inline scripts/build-windows/kcl-doc.go:65]: The judgment here is a little hard. For example, strings.Contains(out.String(), "Package(s) not found")

Comment (zong-zhe) [reply scripts/build-windows/kcl-doc.go]: fixed

**Review by Peefy — APPROVED** [review]
LGTM

**Review by NeverRaR — APPROVED** [review]

**Pull request merged**